pkg/integrations/v4/logs/cfg_test.go

```diff
@@ -1233,9 +1234,10 @@ func TestFBCfgFormat(t *testing.T) {
 				Name:  "record_modifier",
 				Match: "some-folder",
 				Records: map[string]string{
-					"fb.input": "tail",
-					"key1":     "value1",
-					"key2":     "value2",
+					"fb.input":        "tail",
+					"key1":            "value1",
+					"key2":            "value2",
+					"key3 with space": "value3 with space",
```

Comment:
Doing a quick check, it seems that if you specify an attribute with spaces in its _name_, YAML actually accepts it:
![Screenshot 2024-03-11 at 21 39 51](https://github.com/newrelic/infrastructure-agent/assets/6243832/b273c438-d9df-4136-b6e5-919df0293d2d)

I didn't expect that!

Could you please introduce a test [here](https://github.com/newrelic/infrastructure-agent/blob/master/pkg/integrations/v4/logs/loader_test.go#L288-L304) that checks the YAML -> struct conversion, both when having attribute _names_ with whitespaces and attributes _values_ with whitespaces?

Reply:
Hi @jsubirat, Nice suggestion. 
A Test case has been added to check if attribute key and value with spaces are working. Thank you!

Reply:
Amazing thanks!